tests/integration/activity.test.js

```diff
@@ -82,6 +211,76 @@ const test = async app => {
     await tap.equal(res.statusCode, 404)
   })
 
+  await tap.test('Create Read activity with only a selector', async () => {
+    const readActivity = await request(app)
+      .post(`${userUrl}/activity`)
+      .set('Host', 'reader-api.test')
+      .set('Authorization', `Bearer ${token}`)
+      .type(
+        'application/ld+json; profile="https://www.w3.org/ns/activitystreams"'
+      )
+      .send(
+        JSON.stringify({
+          '@context': [
+            'https://www.w3.org/ns/activitystreams',
+            { reader: 'https://rebus.foundation/ns/reader' }
+          ],
+          type: 'Read',
+          object: { type: 'Document', id: document.id },
```

Comment:
actually the context here will be the publication, not the document.
So you don't need to create document for this test.